R/latent_individual.R

```diff
@@ -232,11 +237,24 @@ epidist_stancode.epidist_latent_individual <- function(data,
     scode = epidist_stan_chunk("latent_individual/functions.stan")
   )
 
+  
   family_name <- gsub("latent_", "", family$name)
 
   stanvars_functions[[1]]$scode <- gsub(
     "family", family_name, stanvars_functions[[1]]$scode
   )
+  
+  stanvars_functions[[1]]$scode <- gsub(
+    "dpars_A",
+    paste(paste0("vector ", family$dpars), collapse = ", "),
```

Comment:
isn't dpars a vector of length two?

Reply:
ah I see so dpars_a doesn't do anything?

Reply:
r.e. better yet just leave as is, if I'm correct about `sigma` not existing for some families then we can't do that. So TBD based on resolution of that discussion.

r.e. why not recursively replace, I'm not sure what exactly that would be. I did `dpars_A` and `dpars_B` because you need `vector dpars` in one place and `dpars, ...` in the other.

Reply:
you can if its just us renaming but I am not saying we should do that. 

I'm not sure I understand why isn't vector just written into the code? Why isn it just injecting `mu, sigma` etc?

Reply:
if you could do this it would let us have standard code for injecting dpars and then not have to make it model specific which I think we want

Reply:
> I'm not sure I understand why isn't vector just written into the code? 

There could be an arbitrary number of distributional parameters. So what should be written? `vector dpar1, vector dpar2, vector dpar3, ...`.

> if you could do this it would let us have standard code for injecting dpars

I don't understand how this would make it more standard.

>  and then not have to make it model specific which I think we want

I'm not sure the way the solution here isn't standard code for injecting `dpars`. Is there a way in which this is model specific?